instrumentation/http-url-connection/javaagent/src/test/groovy/HttpUrlConnectionTest.groovy

```diff
@@ -293,6 +294,28 @@ class HttpUrlConnectionTest extends HttpClientTest {
     }
   }
 
+  def "error span"(){
+    def uri = server.address.resolve("/error")
+    when:
+    def url = uri.toURL()
+    runUnderTrace("parent") {
+      doRequest(method, uri, ["is-test-server": "false"])
+    }
+
+    then:
+    def expectedException = new IOException("Server returned HTTP response code: 500 for URL: $url")
+    thrown(IOException)
+    assertTraces(1) {
+      trace(0, 2 + extraClientSpans()) {
+        basicSpan(it, 0, "parent", null, expectedException)
+        clientSpan(it, 1, span(0), method, uri, 500, expectedException)
+      }
```

Comment:
there's no need for extraClientSpans() below, since that's not overridden in this test, but let's keep it, b/c I think we should move this test up to HttpClientTest - I'll open separate issue for that
```suggestion
      trace(0, 3 + extraClientSpans()) {
        basicSpan(it, 0, "parent", null, expectedException)
        clientSpan(it, 1, span(0), method, uri, 500, expectedException)
        serverSpan(it, 2 + extraClientSpans(), span(1 + extraClientSpans()))
      }
```